File: Easy-exercises/Cheer Generator/Cheer_Generator.cpp
#include <iostream>
using namespace std;
int main()
{
    int yard, i = 1;
    cout << "enter value for yard : " << endl;
    cin >> yard;
    if (yard > 10)
    {
        cout << "High Five";
    }
    else if (yard < 1)
    {
        cout << "shh";
    }
    else
    {
        while (i <= yard)
        {
            cout << "Ra!";
            i++;
        }
    }

    return 0;
}

File: Easy-exercises/Cheer Generator/Cheer_Generator.php
<?php
    $i = 1;
    echo ("enter value for yard :\n");
    $yard = (int)readline('');
    if($yard > 10)
    echo ("High Five");
    else if($yard < 1)
    echo ("shh");
    else{
        while($i <= $yard){
            echo ("Ra!");   
            $i++;
        }
    }
?>